src/lib.rs

```diff
@@ -47,9 +54,24 @@ pub struct Args {
     pub vimgrep: bool,
 }
 
+impl Args {
+    pub fn use_paths(&self) -> Vec<PathBuf> {
+        if self.paths.is_empty() {
+            vec![Path::new("./").to_owned()]
```

Comment:
Not that you probably care right now, but do you know if this works on Windows?

Reply:
Ya that had crossed my mind, I'd guess there's some "path"-y stuff that currently wouldn't work but no idea, opened an issue

I should get a Windows laptop that I can dual-boot Linux on since I've been wanting a way to run Linux locally too